### ISSUE TITLE
11.88-1: UnlikelyInteract Wall False Positive

### ISSUE THREAD

**Issue by ghost** [joehot200/AntiAura#149]
**Media:** https://youtu.be/Zg7v96UKQ-k

**Server Version:** Paper version git-Paper-231 (MC: 1.16.3)

**Config:** 
```yaml UnlikelyInteract:
  #Detect if a player attempts to open an inventory (e.g. a chest) through a wall.
  DetectImpossibleInventory: true
  #Maximum legitimate distance from the target object (e.g. a chest) to the players' line of sight ray.
  #Higher values are more lenient, but may allow bypasses through 1-block walls if set too high.
  #Advanced config only.
  MaxInventoryDistance: 1.0
  #If DetectImpossibleInventory is true, block the player from opening inventories through walls.
  BlockImpossibleInventory: true
  #Detect block placements on block faces that cannot be seen?
  DetectImpossiblePlacements: true
  #If DetectImpossiblePlacements, prevent illegitimate block placements.
  BlockImpossiblePlacements: true
  #Amount of hacking attempts to detect player
  #Advanced config only.
  Threshold: 8
  #Violation decrease if a player interacts legitimately
  #Advanced config only.
  SafeInteractDecrease: 1
  #Command to execute when UnlikelyInteract is detected.
  UnlikelyInteractCommand: # 
```

**Reproduction steps:** Shown in the video

**Comment by joehot200** [comment]
Fixed for 11.90

**Comment by ghost** [comment]
Still an issue as of 11.91-1.

**Comment by joehot200** [comment]
Should be fixed for 11.92 please test when 11.92 comes out and close if so.
